src/Facebook/GraphNodes/GraphCoverPhoto.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Copyright 2014 Facebook, Inc.
+ *
+ * You are hereby granted a non-exclusive, worldwide, royalty-free license to
+ * use, copy, modify, and distribute this software in source code or binary
+ * form for use in connection with the web services and APIs provided by
+ * Facebook.
+ *
+ * As with any software that integrates with the Facebook platform, your use
+ * of this software is subject to the Facebook Developer Principles and
+ * Policies [http://developers.facebook.com/policy/]. This copyright notice
+ * shall be included in all copies or substantial portions of the software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ *
+ */
+namespace Facebook\GraphNodes;
+
+/**
+ * Class GraphCoverPhoto
+ *
+ * @package Facebook
+ */
+class GraphCoverPhoto extends GraphObject
+{
+	/**
+     * Returns the id of cover if it exists
+     *
+     * @return int|null
+     */
+    public function getId()
+    {
+        return $this->getProperty('id');
+    }
+	
+	/**
+     * Returns the cover_id of cover if it exists
+     *
+     * @return int|null
+     */
+    public function getCoverId()
+    {
+        return $this->getProperty('cover_id');
```

Comment:
the graph api return both `id` and `cover_id`

Example response:
`{
  "cover": {
    "cover_id": "967028006663070", 
    "offset_x": 0, 
    "offset_y": 20, 
    "source": "https://fbcdn-sphotos-f-a.akamaihd.net/hphotos-ak-xpa1/t31.0-8/s720x720/10353277_967028006663070_5928354600582149744_o.jpg", 
    "id": "967028006663070"
  }, 
  "start_time": "2015-05-02T10:30:00+0300", 
  "id": "1604727873078886"
}`